src/symbolic/symbolic_path_condition.mli

```diff
@@ -0,0 +1,13 @@
+(* SPDX-License-Identifier: AGPL-3.0-or-later *)
+(* Copyright © 2021-2024 OCamlPro *)
+(* Written by the Owi programmers *)
+
+type t
+
+val empty : t
+
+val add : t -> Symbolic_value.vbool -> t
+
+val slice : t -> Symbolic_value.vbool -> Smtml.Expr.Set.t
```

Comment:
The invariant seems to be that slice can only be called when the vbool has already been added to the set.
It should be mentioned, and maybe asserted in the code also

Reply:
It is already asserted (I added a comment in the `.mli` about the invariant).